extensions/catalog/federated-catalog-cache/src/test/java/org/eclipse/dataspaceconnector/catalog/defaults/store/InMemoryFederatedCacheStoreTest.java

```diff
@@ -110,4 +113,23 @@ void queryCacheContainingTwoDistinctAssets_shouldReturnBothContractOffers() {
                 .anySatisfy(co -> assertThat(co.getAsset().getId()).isEqualTo(assetId1))
                 .anySatisfy(co -> assertThat(co.getAsset().getId()).isEqualTo(assetId2));
     }
+
+    @Test
+    void deleteAll() {
```

Comment:
@paullatzelsperger do you think it would be worth writing integration tests for the federated catalog logic? Probably in a separate story as there are none at the moment AFAIS.

Reply:
@marcgs yes that is likely a good idea :)
however, I wanted to get this in quickly, and i-tests or e2e-tests for the catalog are no small feat.

Reply:
Created an issue about: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1279